lib/ruby/lib/alces/packager/config.rb

```diff
@@ -54,6 +54,10 @@ def dbroot(depot)
           File.expand_path(File.join(depot_path(depot),'etc'))
         end
 
+        def last_update_file
+          File.new(method_missing(:last_update_file))
```

Comment:
This would be better, although since you've pointed this out I've realised this method isn't actually being used any more as each repo now has its own last update file (see https://github.com/alces-software/clusterware/blob/f82db8c553ed84fea853608dc9aa800993640694/lib/ruby/lib/alces/packager/repository.rb#L226,L228). I'll remove this method now, thanks for pointing it out.